### PR TITLE
Playbooks - IncidentToContext task and cleanup.

### PR DESCRIPTION
**Pull request by liorkol**
- All versions to -1.
- Added IncidentToContext task in each playbook.
- Removed deprecated IsTitleTask property.
- Removed "clonedfrom" properties.

All yamls upload successfully (tested on build 10161)
fixes https://github.com/demisto/server/issues/3153

### CODE REVIEW

File: Playbooks/playbook-Protectwise_Demo_Playbook.yml [hunk 1]
```diff
@@ -8,6 +8,46 @@ description: Displays information for all events since 2016-06-17 and all observ
   limit .
 tasks:
 - id: "1"
+  taskid: 1bf3408d-bfcc-4030-8406-126c9514ee5b
```
Comment (aviadl) [inline Playbooks/playbook-Protectwise_Demo_Playbook.yml]: Should a playbook with this name be in our system ?
I think that we should only have production playbooks in the content

Comment (aviadl) [inline Playbooks/playbook-Protectwise_Demo_Playbook.yml]: @slavikm @rishibhargava thoughts ?

File: Playbooks/playbook-default.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 id: playbook0
-version: 1
+version: -1
 name: Default
```
Comment (aviadl) [inline Playbooks/playbook-default.yml]: Do we want to change this playbook name to Incident Classifier ?

File: Playbooks/playbook-default.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ tasks:
   type: regular
   task:
     id: ea9ee448-a157-44e5-8266-aba6173516e2
-    version: 1
+    version: -1
```
Comment (aviadl) [inline Playbooks/playbook-default.yml]: Add IncidentToContext task ?

**Comment by aviadl** [comment]
@liorkol  See my comments and decide what you want to do with them

**Comment by slavikm** [comment]
@aviadl Good question. I think that eventually we will have plenty of playbooks and probably have some sort of tags structure like we have in automation scripts. Most playbooks will be there without being directly associated with incident type. Also, it plays to the new page of scheduled playbooks that will be outside of specific incidents.

**Pull request merged**